src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
```

Comment:
Instead of getting winsize back into ioctl.d, I reverted to our original suggestion and made the ttycom and filio imports public. This way, now people can simply include ioctl module and don't need to include the additional ttycom and filio modules. This avoids having to include ioctl in ttycom module, but also keeps the separation of definitions like in the C headers.